runtimes/peregrine/src/benchmarks/mod.rs

```diff
@@ -59,6 +60,9 @@ use pallet_membership as pallet_technical_membership;
 #[allow(unused_imports)]
 use pallet_web3_names as pallet_dot_names;
 
```

Comment:
Why the whitespace?

Reply:
I kinda overcomplicated it: [aaeeeac](https://github.com/KILTprotocol/kilt-node/pull/818/commits/aaeeeacfbf989c2201fde73a478cc47f270fdd48)